ocaml/lambda/translmod.ml

```diff
@@ -1559,7 +1559,7 @@ let toploop_getvalue id =
                   Loc_unknown);
     ap_args=[Lconst(Const_base(
       Const_string (toplevel_name id, Location.none, None)))];
-    ap_result_layout = Lambda.layout_top;
+    ap_result_layout = Lambda.layout_module_field;
```

Comment:
I think that `layout_module_field` is misleading, as in bytecode at least there's no module involved.
My intuition is that this should be an explicit `layout_any_value`, to be updated when we add support for non-value layouts in the toplevel.